src/win/msvc/vc15/VARCem.vcxproj

```diff
@@ -668,6 +668,8 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <LinkIncremental>false</LinkIncremental>
+    <IncludePath>$(ProjectDir)\..\include;$(ProjectDir)\..\..\mingw\include;$(ProjectDir)\..\..\mingw\include\pcap;$(IncludePath)</IncludePath>
+    <LibraryPath>$(ProjectDir)\..\lib\x86;$(VC_LibraryPath_x86);$(WindowsSDK_LibraryPath_x86);$(NETFXKitsDir)Lib\um\x86</LibraryPath>
```

Comment:
the NetFX path is not needed. 
Also, the include/lib paths are set in the global.props file. Please add them there, then they're included in both the debug and release builds